packages/shared-ux/prompt/no_data_views/src/services.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useContext } from 'react';
+
+/**
+ * TODO: `DataView` is a class exported by `src/plugins/data_views/public`.  Since this service
+ * is contained in this package-- and packages can only depend on other packages and never on
+ * plugins-- we have to set this to `unknown`.  If and when `DataView` is exported from a
+ * stateless package, we can remove this.
+ *
+ * @see: https://github.com/elastic/kibana/issues/127695
+ */
+type DataView = unknown;
+
+/**
+ * A subset of the `DataViewEditorOptions` interface relevant to our service and components.
+ *
+ * @see: src/plugins/data_view_editor/public/types.ts
+ */
+interface DataViewEditorOptions {
+  /** Handler to be invoked when the Data View Editor completes a save operation. */
+  onSave: (dataView: DataView) => void;
```

Comment:
I have a small conflicting change coming up, ie. I'm adding a new parameter to the options here. Let's coordinate what goes first, but since it's just one line shouldn't be a stopper for anyone.

Reply:
I can adjust, just wanted to get this drafted before I left for vacation.